src/pages/programDetail/program-detail-view.styled.js

```diff
@@ -1,104 +1,103 @@
-import { styled, withStyles } from "@material-ui/core/styles";
-import Button from "@material-ui/core/Button";
+import Button from '@mui/material/Button';
+import { styled } from '@mui/system';
```

Comment:
should import from `@mui/styles`

Reply:
@mui/styles was deprecated with the release of MUI Core v5 in late 2021. It is not compatible with [React.StrictMode](https://react.dev/reference/react/StrictMode) or React 18+, and it will not be updated.

Reply:
Okay, I see. You need to install the `@mui/system` package though. I don't see it in the package.json
`npm i @mui/system --legacy-peer-deps`